src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,26 @@
 package mate.academy.service;
 
+import static mate.academy.service.UserService.users;
+
+import mate.academy.model.User;
+
 public class AuthenticationService {
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
```

Comment:
use the `UserService` and method `findByEmail`

Reply:
I think I've changed everything needed. Check the result out and give me your feedback, would you kindly :)